src/fr/devsylone/fallenkingdom/display/content/ConstantContent.java

```diff
@@ -55,16 +63,48 @@ public boolean containsAny(@NotNull PlaceHolder... placeHolders) {
         return false;
     }
 
+    /**
+     * Vérifie si ce contenu contient des placeholders PAPI
+     * @return true si des placeholders PAPI sont présents
+     */
+    public boolean containsPapiPlaceholders() {
+        return this.hasPapiPlaceholders;
+    }
+
     @Override
     public @NotNull String format(@NotNull Player player, @NotNull FkPlayer fkPlayer, @NotNull PlaceHolder... placeHolders) {
         if (fkPlayer.useFormattedText()) {
             String replaced = this.value;
+
+            // Traitement des placeholders internes
             for (PlaceHolder placeHolder : this.placeHolders) {
                 replaced = placeHolder.replaceMultiple(replaced, player);
             }
+
+            // Traitement des placeholders PAPI si disponible
+            if (this.hasPapiPlaceholders) {
+                try {
+                    replaced = PlaceholderAPI.setPlaceholders(player, replaced);
+                } catch (Exception e) {
+                    // En cas d'erreur PAPI, on continue sans traiter les placeholders PAPI
+                    Bukkit.getLogger().warning("[FallenKingdom] Erreur lors du traitement des placeholders PAPI: " + e.getMessage());
```

Comment:
```suggestion
                    Fk.getInstance().getLogger().log(Level.WARNING, "Erreur lors du traitement des placeholders PAPI", e);
```
Les plugins ne devraient pas utiliser le logger global. Pour les erreurs extérieures, on est intéressé d'avoir la stacktrace globale, d'où l'exception complète est passée au logger.